### PR TITLE
upgrade chart.js to version 3.2.0

### PR DESCRIPTION
**Pull request by dou-du**
* Upgrade the chart.js library to v3.2.0.

### CODE REVIEW

File: js/lib/bandstructure.js [hunk 1]
```diff
@@ -37,7 +37,7 @@
 // Utility 'zip' function analogous to python's, from
 // https://stackoverflow.com/questions/4856717
 
-const Chart = require('chart.js');
+require('chart.js/dist/chart');
```
Comment (unkcpz) [inline js/lib/bandstructure.js:40]: Could you explain why the variable is not used by still need to initialize? I search the file and file line 513 has `Chart`.

**Review by unkcpz — CHANGES_REQUESTED** [review]
Hi @dou-du, thanks! I am not very familiar with JS, hope it is not a silly question ;)

File: js/lib/bandstructure.js (hunk 1 above)
Comment (dou-du) [reply js/lib/bandstructure.js:40]: For version 3.x, the import method has been changed. Yes, we are still need to use Chart function. However, we do not need to init a const.

Comment (unkcpz) [reply js/lib/bandstructure.js:40]: I see, so once import module by `require('chart.js')`, `Chart` function in line `513` is able to be used, correct?

Comment (dou-du) [reply js/lib/bandstructure.js:40]: require('chart.js/dist/chart') will work.

**Review by unkcpz — APPROVED** [review]

**Comment by dou-du** [comment]
> Hi @dou-du, thanks! I am not very familiar with JS, hope it is not a silly question ;)

Not at all, I also spent some time to figure out there is some change for the importing of the Chart. :)

**Pull request merged**